internal/policy/container/has_modified_files.go

```diff
@@ -180,69 +202,242 @@ func (p HasModifiedFilesCheck) Metadata() check.Metadata {
 	}
 }
 
-// Untar takes a destination path and a reader; a tar reader loops over the tarfile
-// creating the file structure at 'dst' along the way, and writing any files
-func untar(pathChan chan<- string, r io.Reader) error {
-	tr := tar.NewReader(r)
+func extractPackageNameVersionRelease(pkgList []*rpmdb.PackageInfo) map[string]struct{} {
+	pkgNameList := make(map[string]struct{}, len(pkgList))
+	for _, pkg := range pkgList {
+		pkgNameList[fmt.Sprintf("%s-%s-%s", pkg.Name, pkg.Version, pkg.Release)] = struct{}{}
+	}
+	return pkgNameList
+}
 
-	for {
-		header, err := tr.Next()
+// FindRPMDB attempts to extract a valid RPMDB from layers in the order
```

Comment:
```suggestion
// findRPMDB attempts to extract a valid RPMDB from layers in the order
```
Nit

Reply:
This is like this in other places too, not sure if it's the linter doing that, or if that's how we want it.

Reply:
I changed it to keep the comment, but also match the actual function name.